### PR TITLE
Removing pending in test and getting the code to work correctly.

### PR DESCRIPTION
**Pull request by cam156**
I needed to set the creator to empty to make the spec pass, which makes sense since the depositor is not going to be the creator of anything he is archiving.

### CODE REVIEW

File: app/actors/actor.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@ class Actor < Sufia::GenericFile::Actor
   def create_metadata(batch_id)
     super
     accession = Accession.find(batch_id)
+    generic_file.creator = ""
```
Comment (awead) [inline app/actors/actor.rb]: can you explain more about this?

Comment (cam156) [inline app/actors/actor.rb]: @awead see my comment on the pull request.  I explained it.

**Comment by awead** [comment]
We aren't seeing any failures when we upload files, so I'm confused as to why we would need to set creator to empty just to get a test to pass. Also doesn't this prevent anyone from setting the creator to a value since this would just override it?

**Comment by cam156** [comment]
@awead This was a requirement of ArchiveSphere, that the creator was not set based on the depositor, since the depositor in an archival system is almost never the creator.  (I stated that in the PR description)  I will admit the test for this seems a bit hidden, but I would believe it is still a valid requirement and that we should honor it. 

 The test clearly outlines that the creator should not be set.  Our code should pass all existing tests to be a valid upgrade/refactor.  Unless the the requirement changed, which I doubt it would have in this case, but if you and Ben discussed this change please correct me.

Take a look at the files in production (https://archivesphere.dlt.psu.edu/files/2n49t181k)  the creator metadata is not set like it is in ScholarSphere.

**Comment by awead** [comment]
@cam156 Ok, that makes sense then. I didn't see the requirement explained in any of the tests, but perhaps I missed it. Maybe some comments in Actor could explain that? Also, by doing this in the actor we're ensuring this happens for any file uploaded, no matter if it's in-line or via a job?

**Pull request merged**